benchmarks/src/main/java/io/druid/benchmark/DimensionPredicateFilterBenchmark.java

```diff
@@ -95,6 +96,19 @@ public boolean applyLong(long input)
             }
           };
         }
+
+        @Override
+        public DruidFloatPredicate makeFloatPredicate()
+        {
+          return new DruidFloatPredicate()
```

Comment:
Using singletons for these false predicates now, in DruidLongPredicate/DruidFloatPredicate